ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -581,22 +586,25 @@ private[platform] object JdbcLedgerDao {
   ): LedgerReadDao =
     new MeteredLedgerReadDao(
       new JdbcLedgerDao(
-        dbSupport.dbDispatcher,
-        servicesExecutionContext,
-        eventsPageSize,
-        eventsProcessingParallelism,
-        acsIdPageSize,
-        acsIdPageBufferSize,
-        acsIdFetchingParallelism,
-        acsContractFetchingParallelism,
-        acsGlobalParallelism,
-        metrics,
-        lfValueTranslationCache,
-        enricher,
-        SequentialWriteDao.noop,
-        participantId,
-        dbSupport.storageBackendFactory.readStorageBackend(ledgerEndCache, stringInterning),
-        dbSupport.storageBackendFactory.createParameterStorageBackend,
+        dbDispatcher = dbSupport.dbDispatcher,
```

Comment:
for the 100th time, please, @pbatko-da keep not necessary refactorings out of PRs with lot of meat :pray: 
I understand why you are doing it. It is great that you care! Please care about it in another PR!!

Reply:
Ok, changed